pkg/streamformatter/streamformatter.go

```diff
@@ -170,3 +188,22 @@ func (sf *StderrFormatter) Write(buf []byte) (int, error) {
 	}
 	return len(buf), err
 }
+
+// AuxFormatter is a streamFormatter that writes aux progress messages
+type AuxFormatter struct {
+	io.Writer
+	*StreamFormatter
```

Comment:
`StreamFormatter` is being removed in #32949. It needed a lot of cleanup, it was poorly structured.

Using a struct for a boolean field that is never read (because it always uses json), is not a good reason to use this struct.